plugins/generator-1.20.4/neoforge-1.20.4/templates/livingentity/livingentity.java.ftl

```diff
@@ -208,18 +208,18 @@ public class ${name}Entity extends ${extendsClass} <#if data.ranged>implements R
     </#if>
 
 	<#if data.mobModelName == "Biped">
-	@Override public float getMyRidingOffset(Entity entity) {
+	@Override protected float ridingOffset(Entity entity) {

```

Comment:
`getMyRidingOffset(Entity)` returns scaled offset for living entities and static offset for other ones. Keep this?

Reply:
Porting is really not suitable for functional changes. Please make sure this PR is only porting and does not do other changes to how things behave. Porting should initially try to mimic existing generator behavior as much as realistically possible.

Reply:
Strangely it looks like changing one method to another here changes nothing in-game.

Reply:
Given they refactored the previous method, this one is more similar to the original one functionally. I'll keep it.